modules/statics/src/coins.ts

```diff
@@ -832,6 +832,20 @@ export const coins = CoinMap.fromCoins([
   ofcerc20('ofcusdt', 'Tether', 6, UnderlyingAsset.USDT),
   ofcerc20('ofcxsushi', 'xSUSHI', 18, UnderlyingAsset.XSUSHI),
   ofcerc20('ofcyfi', 'Yearn.Finance', 18, UnderlyingAsset.YFI),
+  ofcerc20('ofcamp', 'amp Token', 18, UnderlyingAsset.AMP),
```

Comment:
```suggestion
  ofcerc20('ofcamp', 'Amp Token', 18, UnderlyingAsset.AMP),
```
Should this be Capital Case?